Web of Science Tagged.js

```diff
@@ -229,6 +229,11 @@ function doImport(text) {
 	while ((rawLine = Zotero.read()) !== false) {    // until EOF
 		//Z.debug("line: " + rawLine);
 		let split = rawLine.match(/^([A-Z0-9]{2})\s(?:([^\n]*))?/);
+		// EF is equivalent to the end of file.
+		if (/^EF\s*/.test(rawLine)) {
```

Comment:
The `\s*` isn't doing anything here.

Reply:
At least assuming every line is guaranteed to start with a tag… How are multi-line fields (e.g., multi-line notes) handled?